packages/components/tabs/src/tab-panel.tsx

```diff
@@ -14,19 +14,24 @@ export interface TabPanelProps extends HTMLUIProps {}
 
 export const TabPanel = forwardRef<TabPanelProps, "div">(
   ({ id, className, children, ...rest }, ref) => {
-    const { isLazy: enabled, lazyBehavior: mode, styles } = useTabsContext()
+    const {
+      isLazy: isLazyEnabled,
```

Comment:
Don't change.

Reply:
Hi @hirotomoyamada , since you suggested me to remove the `isLazy` value from the tabs context provider, and it's now using `lazy`, so in the `tab-panel`, I will need to make this change otherwise it failed on the test case below:

```diff
- const { isLazy: enabled, lazyBehavior: mode, styles } = useTabsContext()
+ const { lazy: enabled, lazyBehavior: mode, styles } = useTabsContext()
```

https://github.com/yamada-ui/yamada-ui/blob/433f6cadfe42ed58acf116ba4c6f08f41fca6761/packages/components/tabs/tests/tabs.test.tsx#L55-L78

Reply:
@nathanlao 

Hi, That's right! Thanks for the change!